compiler/testData/diagnostics/tests/multiplatform/headerClass/nestedClasses.kt

```diff
@@ -1,14 +1,22 @@
-// FIR_IDENTICAL
```

Comment:
Note that technically they're identical, but somehow other tags with `{JVM}` bother.